ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -907,3 +955,164 @@ objects:
           fi
           sleep 60 & wait
         done
+
+    # The setup-packet and test containers need libnns_wrapper to use ssh
+    # TODO(derekh): investigate if it can be added to that container images
+    - name: nss-wrapper-hack
```

Comment:
Am I right to think we would be running this pod on every e2e install, whether it is needed or not?

Reply:
Yes, it copies a bunch of files to a shared folder since they are needed by  the setup-packet container